src/pages/home/sub-components/categories.js

```diff
@@ -0,0 +1,18 @@
+import { categoryList } from "./data"
+import styles from "../home.module.scss"
+import { Image } from "react-felix-ui"
+const Catagories = () => {
+    return (
+        categoryList.map((item, i) => {
+            return (
+                <a key={i} href="#" className={styles.item}>
```

Comment:
Nitpicking: Better not to use the index as a key.
Here is a great article you can refer to why using the index as the key is anti-pattern.:https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Using index just for now. will change after I connect it with the backend.